packages/terra-date-input/src/DateInput.jsx

```diff
@@ -382,11 +382,12 @@ class DateInput extends React.Component {
     }
 
     const inputValue = event.target.value;
-    const maxValue = 9999;
+    const maxValue = 2100;
+    const minValue = 1900;
```

Comment:
Can we move these to DateInputUtil.js similar to [Date-picker](https://github.com/cerner/terra-framework/blob/6f0bb1db7b670b5d900049295e73d03713ed6059/packages/terra-date-picker/src/DateUtil.js#L252-L253) so that same can be used in DateInputUtils instead of using year values '2100' in multiple places.

Reply:
Fixed 6f09098